### DISCUSSION TITLE
Delete Users

### DISCUSSION THREAD

**Discussion by mbiolsi** [docmost/docmost#539] [Ideas]
Can we please see an option to remove/delete a user or to make them inactive?

**Comment by Philipinho** [comment]
Some work to be done on this PR https://github.com/docmost/docmost/pull/373.

**Comment by BeanHubbleday** [comment]
Yes, definitely needed. We've got a leaver and now our internal documentation is wide open.

**Comment by Wasca** [comment]
Any updates to this request. I was just testing the product and added a test user not realizing you cannot delete a user.
